spec/std/va_list_spec.cr

```diff
@@ -1,4 +1,4 @@
-{% skip_file if flag?(:win32) %}
```

Comment:
Isn't this spec being run today in some ararch64 platforms? Why exclude it?

Reply:
It is by sheer coincidence that it doesn't break on some AArch64 targets. To this date I don't think the situation has ever improved

Reply:
You mean because of LLVM, or something Crystal specific? In the first case, maybe it got fixed (I couldn't find any open bug there)? In the second case, we should be tracking it. It doesn't work there in your M2?

Reply:
I _think_ we could implement the whole `VaList` thing ourselves if we want to, similar to Rust (https://github.com/crystal-lang/crystal/pull/9422#issuecomment-667899436); until then we should not assume that `VaList` is supported on any AArch64 targets at all. (This refers to the whole `VaList` type and not just the `@[Primitive(:va_arg)]` part.)